src/v/resource_mgmt/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+v_cc_library(
```

Comment:
Needed a new lib b/c all existing files in here were header only, but I am anti putting everything in the header.